apps/web/src/pages/auth/components/LoginForm.tsx

```diff
@@ -130,7 +130,13 @@ export function LoginForm({ email, invitationToken }: LoginFormProps) {
     try {
       const response = await mutateAsync(itemData);
       const token = (response as any).token;
-      login(token);
+      await login(token);
+
+      if (isFromVercel) {
+        startVercelSetup();
+
+        return;
+      }
```

Comment:
fixed the issue where the vercel configuration setup was not kicking in after login